templates/index.html

```diff
@@ -11,7 +12,7 @@
         </nav>
     </header>
     <div>
-      <wpt-results></wpt-results>
+      <wpt-results test-runs="{{ . }}"></wpt-results>
```

Comment:
The dot refers to the top-level object in the template - see the last argument in `templates.ExecuteTemplate`:
https://github.com/GoogleChrome/wptdashboard/pull/10/files#diff-eef986022544a4cde28537615d5c6768R50

For example if the top-level object in the template were a struct, you'd write `{{ .Property }}` to access `struct.Property`.